tests/sample_rule_files/free-standing/error/alert.rule

```diff
@@ -0,0 +1,20 @@
+groups:
```

Comment:
Is the subdir needed? I.e. would `free-standing/alert_with_syntax_error_in_expr.rule` work?

Reply:
I believe it's better to have them in a separate directory so we can use cos-config in itests:

```
        ops_test.model.deploy(
            "ch:cos-configuration-k8s",
            application_name=rules_app3,
            channel="edge",
            config={
                "git_repo": "https://github.com/canonical/loki-k8s-operator",
                "git_branch": "main",
                "loki_alert_rules_path": "tests/sample_rule_files/free-standing/error",
            },
        ),
    )
```